avaframe/com1DFA/com1DFA.py

```diff
@@ -881,7 +892,14 @@ def initializeParticles(cfg, releaseLine, dem, inputSimLines='', logName='', rel
     areaRaster = dem['areaRaster']
 
     # get the initialization method used
-    massPerPart, nPPK = com1DFATools. getPartInitMethod(cfg, csz)
+    if len(relThField) != 0:
+        relThForPart = np.amax(relThField)
+    elif cfg.getboolean('relThFromShp'):
+        relThForPart = np.amax(np.asarray(releaseLine['thickness'], dtype=float))
```

Comment:
This should be tested

Reply:
moved this to a separate function and included a pytest